src/containers/Programs/index.tsx

```diff
@@ -180,8 +180,8 @@ const Programs = ({
   };
 
   useEffect(() => {
-    if (information) setStateToRender(information);
-  }, [information, stateToRender]);
+    setStateToRender(information);
+  }, [information]);
```

Comment:
Are you sure we can remove the condition inside useEffect?

Reply:
The dependency array triggers a re-render. which means that we do not need to use if check.